app/Mail/PwnedPasswordNotification.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Proto\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+use Proto\Models\User;
+
+class PwnedPasswordNotification extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    public $name;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct(User $user)
+    {
+        $this->name = $user->name;
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this
+            ->subject('IMPORTANT SECURITY NOTICE: The password you use with S.A. Proto was exposed in a data breach.')
```

Comment:
Maybe change the subject to something that immediately makes clear that this wasn't a data breach of Proto's data.

Reply:
I'm not sure if that is the right approach. It is stated very clearly in the e-mail, and perhaps this subject is a better trigger to read the e-mail.

Reply:
Good point, agreed :)